bitnami/kubeapps/values.yaml

```diff
@@ -27,13 +27,13 @@ nameOverride:
 ## @param fullnameOverride String to fully override common.names.fullname
 ##
 fullnameOverride:
-## @param commonLabels Labels to add to all deployed objects
+## @param commonLabels [object] Labels to add to all deployed objects
```

Comment:
I would like @miguelaeh to confirm this BUT I don't think you need the `[object]` the modifier when the parameter is already set by default to the empty object `{}`. Same applies to arrays that are set by default to `[]`.
Could you please try removing it and checking if there's any difference?

Reply:
> Could you please try removing it and checking if there's any difference?

Great, they are not required when using empty objects. No difference when removing them

Reply:
That's odd: I required them when running the readmenator tool otherwise I was getting `undefined` for all those empty params. Perhaps it was the (latest) version that I was using? Not sure.